src/lib/radio/radio.ts

```diff
@@ -274,7 +289,7 @@ export class MdRadioButton implements OnInit {
 
   @Input()
   get disableRipple(): boolean { return this._disableRipple; }
-  set disableRipple(value) { this._disableRipple = coerceBooleanProperty(value); }
+  set disableRipple(value) { this._disableRipple = (value); }
```

Comment:
hm. I ran `npm test` after changes this one got away. sorry. changed and pushed again. thanks